contracts/UniswapV3Staker.sol

```diff
@@ -131,14 +130,16 @@ contract UniswapV3Staker is
 
         Incentive memory incentive = incentives[key];
         require(incentive.rewardToken != address(0), 'INVALID_INCENTIVE');
-        delete incentives[key];
 
         // TODO: check for possible failures
-        IERC20Minimal(params.rewardToken).transfer(
+        TransferHelper.safeTransfer(
+            params.rewardToken,
             msg.sender,
             incentive.totalRewardUnclaimed
         );
 
+        delete incentives[key];
```

Comment:
I think we should leave this above in case the token tries to call back into the function and re-transfer.